src/components/Chat/Addons.tsx

```diff
@@ -195,7 +187,7 @@ export const Addons = ({
           )}
           <div>
             <button
-              className="mt-3 inline text-left text-blue-500"
```

Comment:
Earlier for color was lighter in light theme. Is it part of color changing?
![image](https://github.com/epam/ai-dial-chat/assets/20163971/cef00852-61c7-4b96-a82b-eb8d6c41ddba)